
# Global frame

geometry_msgs/Point position

float64 width
float64 inclination
